vue/test/setup.js

```diff
@@ -1,4 +1,3 @@
-require("@babel/polyfill");

```

Comment:
This is the kind of thing that must go in a separate PR, even though it's a pain to all of us :(